src/lib/utils.js

```diff
@@ -380,8 +380,17 @@ export const ErrorCode = {
   },
 };
 
-export function parseConfigFile(configPath: string): Object {
-  return YAML.safeLoad(fs.readFileSync(configPath, CHARACTER_ENCODING.UTF8));
+function parseConfigFile(configPath: string) {
```

Comment:
`export` is missing.

Reply:
My mistake, fixed.

Reply:
```
/home/circleci/verdaccio/src/lib/utils.js
  386:10  error  Insert `·`  prettier/prettier
  389:12  error  Insert `·`  prettier/prettier
```
still prettiers yells, just that little thing please :-)

Reply:
fixed